applications/chats/admin.py

```diff
@@ -3,5 +3,32 @@
 from .models import Message, Room
 
 
-admin.site.register(Room)
-admin.site.register(Message)
+class MessagesInline(admin.TabularInline):
+    model = Message
+    fields = ('writer', 'content', 'created_at', 'is_read', )
+    readonly_fields = ('writer', 'content', 'created_at', 'is_read', )
+
+    def has_add_permission(self, *args):
+        return False
+
+
+class RoomAdmin(admin.ModelAdmin):
+    list_display = ('id', 'topic', 'members_names', 'created_at', )
+    fields = ('topic', 'members', 'created_at', )
+    filter_horizontal = ('members', )
+    readonly_fields = ('created_at', )
+    inlines = (MessagesInline, )
+
+    def members_names(self, obj):
+        members = obj.members.all()
+        return f'\'{members.first()}\' and \'{members.last()}\''
```

Comment:
What do you want to return here?

Reply:
I want to display in this column the members of the room in the format `'Bruno' and 'Agustin'`

Reply:
I think in this case is better to use double quotes to avoid escaping characters.
```suggestion
        return f"'{members.first()}' and '{members.last()}'"
```